### PR TITLE
Code coverage plugin, Import Jobs tests, Fixes on tests, Code coverag…

### PR DESCRIPTION
**Pull request by ascuccimarra**
…e on 21%

### CODE REVIEW

File: test/integration/simplecare/UserImportServiceSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    def setup() {
+    }
+
+    def cleanup() {
+    }
+
+    void "test import job"() {
+        when:
+
+            def f = new GrailsMockMultipartFile('file.csv', 'test@testuser.com,Jane,Doe,1/1/1970,Tarmac,M,PATIENT,,,'.bytes)
```
Comment (ppazos) [inline test/integration/simplecare/UserImportServiceSpec.groovy]: @ascuccimarra please add a null tests (i.e. no input file, and input file empty)

File: test/integration/simplecare/UserServiceSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        then:
+            assert p.firstName == 'lala'
+    }
+
+    void "test save healthcare provider"() {
+        when:
+        def p = HealthcareProvider.findByUser(User.findByUsername('todd'))
+        p.firstName = 'lala'
+        userService.saveDoctorProfile(p)
+        p = HealthcareProvider.findByUser(User.findByUsername('todd'))
```
Comment (ppazos) [inline test/integration/simplecare/UserServiceSpec.groovy]: @ascuccimarra for clearness user different variable names

**Comment by ppazos** [comment]
@ascuccimarra because the tests are complex, put a little comment above the test with the general idea of what the test does and the expected results. Are you updating the test case spreadsheet?

**Comment by ascuccimarra** [comment]
@ppazos made the changes. I wasn't updating the spreadsheet, I was basically trying to bump up code coverage.

**Comment by ascuccimarra** [comment]
@ppazos did you have time to recheck this?

**Comment by ppazos** [comment]
+1 @ascuccimarra

**Pull request merged**